### DISCUSSION TITLE
Trivy k8s doesn't support templates

### DISCUSSION THREAD

**Discussion by ssilbory** [aquasecurity/trivy#5556] [Q&A]
### Description

When I run _trivy k8s pods  -n default -o report.html --format=template --template='@contrib/html.tpl_  I get a usage statement with the following at the end.

`invalid argument "template" for "-f, --format" flag: must be one of ["table" "json" "cyclonedx"]`

The help output above this seems to imply that you can use templates.
`  -t, --template string        output template`

This works with trivy image, but there is nothing in the online documentation that indicates this does not work for k8s.

### Desired Behavior

Either for templates to work or for the documentation to not tell me it works.

### Actual Behavior

`invalid argument "template" for "-f, --format" flag: must be one of ["table" "json" "cyclonedx"]`

### Reproduction Steps

```bash
trivy k8s pods  -n default -o report.html --format=template --template='@contrib/html.tpl
```

### Target

Kubernetes

### Scanner

Vulnerability

### Output Format

Template

### Mode

Standalone

### Debug Output

```bash
Debug doesn't work as the command just outputs usage and exits.
```

### Operating System

ubuntu linux

### Version

```bash
Version: 0.46.1
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-11-10 18:11:17.877917 +0000 UTC
  NextUpdate: 2023-11-11 00:11:17.87791662 +0000 UTC
  DownloadedAt: 2023-11-10 20:55:38.015183487 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2023-11-03 00:53:41.190853802 +0000 UTC
  NextUpdate: 2023-11-06 00:53:41.190853402 +0000 UTC
  DownloadedAt: 2023-11-03 22:26:05.15041779 +0000 UTC
Policy Bundle:
  Digest: sha256:1df8ade71efc830877ca3b1130f83e0c6368e3a45b0d4c0f0418955501644054
  DownloadedAt: 2023-11-10 20:55:38.107472649 +0000 UTC
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by fatihtokus** [comment]
Hi @ssilbory ,

I have an open-source trivy plugin(https://github.com/fatihtokus/scan2html) that creates a well-formated filterable/sortable html report for any scan result of trivy. The usage is very simple and I guess it does what you are looking for, please have a look:

Install the plugin: $ trivy plugin install github.com/fatihtokus/scan2html
$ trivy scan2html k8s --report=all cluster result.html

![image](https://github.com/aquasecurity/trivy/assets/19647150/98ee855e-22fc-4f13-a91d-603001ae6435)

**Comment by DmitriyLewen** [comment]
Hello @ssilbory 

Trivy doesn't currently support templates for `k8s` mode.

Regards, Dmitriy
